wcfsetup/install/files/lib/system/clipboard/action/UserClipboardAction.class.php

```diff
@@ -227,13 +270,14 @@ protected function validateMerge() {
 	 */
 	protected function validateResendActivationMail() {
 		// check permissions
-		if (!WCF::getSession()->getPermission('admin.user.canEnableUser') || REGISTER_ACTIVATION_METHOD != 1) {
+		if (!WCF::getSession()->getPermission('admin.user.canEnableUser') || !(REGISTER_ACTIVATION_METHOD & User::REGISTER_ACTIVATION_USER)) {
 			return [];
 		}
 		
 		$userIDs = [];
 		foreach ($this->objects as $user) {
-			if ($user->activationCode) $userIDs[] = $user->userID;
+			/** @var User $user */
+			if ($user->requiresEmailActivation()) $userIDs[] = $user->userID;
```

Comment:
Should this rather be: `canEmailConfirm`?

Reply:
Yeah, that would actually make more sense.